keps/prod-readiness/sig-scheduling/5254.yaml

```
@@ -0,0 +1,3 @@
+kep-number: 5254
+alpha:
+  approver: "@johnbelamaric"
```

```suggestion
  approver: "@deads2k"
```

I'm not sure the context but david is assigned already in the PRR board.